data/human/republic dump intro.txt

```diff
@@ -0,0 +1,1034 @@
+
+mission "Joining the Navy"
+	autosave
+	name "Navy Candidate"
+	description "Destroy the pirate vessel named <npc> near the <system> system and return to <planet> for an opportunity to join the Republic Navy."
+	source
+		government "Republic"
+	destination "Geminus"
+	on fail
+		dialog `You have failed an essential Navy mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	to offer
+		has "event: war begins"
+		has "event: enlistment begins"
+#		not "event: naval employment earned"
+		has "enlistment announced: offered"
+		not "chosen sides"
+	on offer
+		conversation
+			`Your ship's communicator lights with an official announcement from the Republic:`
+			``
+			`	"Hello, Captain <last>," the message begins, "as of now, the Republic Navy is seeking captains skilled in combat to serve in the Navy. However, to discourage people hoping for handouts, applicants need to prove that they want to serve in the Navy. Recently, we've been having issues with a pirate vessel called the <npc> - our intel reports that it was spotted skulking around the <system> system. Destroy the <npc>, and return to <destination> to be formally accepted into the Navy."`
+			choice
+				`	I'll take on that ship.`
+					goto jobAccept
+				`	Sorry, but I have no desire to join the Navy at this time.`
+					defer
+			label jobAccept
+			`	"We look forward to seeing you on <destination> once the mission is over," the message reads. "Good luck!"`
+					accept
+
+	on accept
+		log "Accepted an enlistment offer from the Republic Navy."
+	
+	npc kill
+		personality heroic staying nemesis target
+		government "Bounty"
+		system "Castor"
+		ship "Raven" "Blackfury"
+				
+	on complete
+		"reputation: Republic" += 10
+		"salary: Navy" = 500
+		set "chosen sides"
+		set "joined the Navy"
+		set "license: Navy"
+#		event "naval employment earned"
+		event "Navy Painted"
+		log "Was able to destroy the pirate ship and prove my worth to the Navy. The Navy has inducted me as an Ensign and given me a small salary."
+		conversation
+			`As you cycle the hatch to exit the ship, you are greeted by two Navy personnel flanking the entryway, armed with rifles. Between them, a Navy officer stands with his arms folded in front of him.`
+			``
+			`	"I assume you are <first> <last>?" he asks. You nod, saluting with your right hand. "Yes sir. And you are...?"`
+			`	"Colonel Marcus Howell," he replies. "Well met, Captain <last> - or rather, Ensign <last>."`
+			`	You nod, still saluting. "Thank you, sir. I hope to be of great service to the Republic."`
+			`	"I have a feeling you'll be just the pilot we need, <last>," he states while making a slight bounce on the tips of his feet, smiling. "The speed and skill with which your ship, the <ship>, destroyed that Raven, in addition to your past endeavors, proves you are both serious about joining the Navy... and it's ready for battle."`
```

Comment:
"proves you are both serious about joining the Navy... and it's ready for battle."
You're both serious and... what, exactly? You're talking about first last and suddenly, that derailed.